### PR TITLE
meson.build: add powerpc arch

### PR DESCRIPTION
**Pull request by barracuda156**
Add PowerPC to valid archs.

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -101,6 +101,12 @@ if ['linux', 'android', 'ios', 'darwin'].contains(system)
   elif cpu_family == 'riscv64'
     # We dont't have riscv64-specific optimization for now.
     asm_format = asm_format64
+  elif cpu_family == 'ppc'
+    # We dont't have powerpc-specific optimization for now.
```
Comment (ryandesign) [inline meson.build:105]:
```suggestion
    # We don't have powerpc-specific optimization for now.
```

File: meson.build [hunk 2]
```diff
@@ -101,6 +101,12 @@ if ['linux', 'android', 'ios', 'darwin'].contains(system)
...
+    asm_format = asm_format32
+  elif cpu_family == 'ppc64'
+    # We dont't have powerpc64-specific optimization for now.
```
Comment (ryandesign) [inline meson.build:108]:
```suggestion
    # We don't have powerpc64-specific optimization for now.
```

File: meson.build [hunk 3]
```diff
@@ -101,6 +101,12 @@ if ['linux', 'android', 'ios', 'darwin'].contains(system)
   elif cpu_family == 'riscv64'
     # We dont't have riscv64-specific optimization for now.
```
Comment (ryandesign) [inline meson.build:102]:
```suggestion
    # We don't have riscv64-specific optimization for now.
```

File: meson.build (hunk 1 above)
Comment (barracuda156) [reply meson.build:105]: Thanks, Ryan, I just copy-pasted a note about RISC-V without checking the spelling. It is worth fixing that line perhaps as well then.

**Comment by barracuda156** [comment]
@1svdgfg Could you please review this as well?

**Review by 1svdgfg — APPROVED** [review]

**Pull request merged**